DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tb_utils.py

```diff
@@ -551,3 +552,23 @@ async def tb_device_tcpdump(device, interface, options, count_only=False, timeou
 
     else:
         return out
+
+
+async def tb_get_qualified_ports(device, ports, speed, duplex):
+
+    # Check dut media mode name. If it supports the speed parameter
+    if device.media_mode == "mixed":
+        raise Exception("Mixed mode is not supported")
+    if device.media_mode == "copper" and speed == 10000:
+        pytest.skip(reason=f"Can not run test in device {device.host_name}  with the speed: {speed}")
```

Comment:
We should not skip the test here, better to raise another exception and catch it in the test

Reply:
I asked about this on a call. Anton suggested to skip the test